README.md

```diff
@@ -34,13 +34,13 @@ launch the command-line in the appropriate virtual environment.
 
 For development in the  `pyanura` repository you should setup a virtual environment in which you will install the dependencies of `pyanura` but not the `pyanura` package itself.
 
-Assuming you have activated a suitable a virtual environment, install the dendencies as follows:
+Assuming you have activated a suitable a virtual environment, install the dependencies as follows:
 
-    pip3 install -r requirements.txt
+    pip3 install .[cli]
```

Comment:
Again, this will install pyanura in the current venv. Will that not create an import conflict when trying to run pyanura code?